src/main/java/teammates/common/datatransfer/FeedbackQuestionAttributes.java

```diff
@@ -287,7 +287,7 @@ public boolean isResponseVisibleTo(FeedbackParticipantType userType) {
      * @param newAttributes
      * @return
      */
-    public boolean isChangesRequiresResponseDeletion(FeedbackQuestionAttributes newAttributes) {
+    public boolean doChangesRequireResponseDeletion(FeedbackQuestionAttributes newAttributes) {
```

Comment:
While it sounds 'boolean' when used e.g. `if(doChangesRequiresResponseDeletetion())`, 'do' can be interpreted as a verb. `isResponseDeletionRequiredForChanges()`?